<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grafo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/Arquivos/PUC.png</normaloff>:/Arquivos/PUC.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_Vertice">
      <property name="title">
       <string>Vertice</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_VerticeDescricao">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Descricao:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_IncluirVertice">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>25</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_IncluirVertice">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Incluir</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Arquivos/include.png</normalon>
          </iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="5">
     <widget class="QStackedWidget" name="stackedWidget_Saida">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="label_Titulo_Lista">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>LISTA DE ADJACENCIA</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="textEdit_Saida">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QTextEdit" name="textEdit_Saida_Matriz">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Titulo_Matriz">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>MATRIZ DE ADJACENCIA</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_Aresta">
      <property name="title">
       <string>Aresta</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_ArestaOrigem">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Origem:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_ArestaDestino">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Destino:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBox_VerticeOrigem">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_VerticeDestino">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_ArestaPeso">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Peso:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="spinBox_ArestaPeso">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkBox_isDirected">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Direcionada</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_IncluirAresta">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Incluir</string>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/Arquivos/include.png</disabledoff>
            </iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>pushButton_SeeAresta</zorder>
      <zorder>label_ArestaOrigem</zorder>
      <zorder>label_ArestaDestino</zorder>
      <zorder>comboBox_VerticeOrigem</zorder>
      <zorder>comboBox_VerticeDestino</zorder>
      <zorder>label_ArestaPeso</zorder>
      <zorder>spinBox_ArestaPeso</zorder>
      <zorder>checkBox_isDirected</zorder>
      <zorder></zorder>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="Line" name="line_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="4">
     <widget class="QGroupBox" name="groupBox_Opcoes">
      <property name="title">
       <string>Opções</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_Opcoes">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_PrintMatriz">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Ver Matriz</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_LimparVertices">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Limpar</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/Arquivos/clear.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_Print_Path">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Print Path</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>pushButton_PrintMatriz</zorder>
          <zorder>groupBox_Opcoes</zorder>
          <zorder>pushButton_Print_Path</zorder>
          <zorder>pushButton_LimparVertices</zorder>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QLabel" name="label_PrintPath_Destino">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Destino</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBox_PrintPath_Origem">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_PrintPath_Origem">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Origem</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_PrintPath_Destino">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_PrintPath_Realizar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Print Path</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
      <zorder>pushButton_IncluirAresta</zorder>
      <zorder>stackedWidget_Opcoes</zorder>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_Resultado_PrintPath">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>RESULTADO DO PRINT PATH</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QTextEdit" name="textEdit_Print_Path">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>97</width>
        <height>87</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionCadastrar_Matriz"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Arquivos/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/save.png</normaloff>:/Arquivos/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCadastrar_Matriz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/grafo.png</normaloff>:/Arquivos/grafo.png</iconset>
   </property>
   <property name="text">
    <string>Cadastrar Matriz</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
